src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --color-hover: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
You should use fallback fonts. This is a good practice to ensure that your website has a consistent appearance across different browsers. If the main font (Roboto in your case) fails to load for some reason, the browser will use the fallback font. It could be a generic font family like `sans-serif` or `serif`. You've done this correctly.